docs/release_notes.adoc

```diff
@@ -17,6 +17,18 @@ With the new behavior, you can have a `Mock` or `Spy` return the same value as a
 subscriber.receive(_) >> _
 ----
 
+== 2.0-M3 (UNRELEASED)
```

Comment:
why is this duplicating some of the stuff from above?

Reply:
As I mentioned in the PR description only the [last commit](https://github.com/spockframework/spock/pull/1128/commits/8c50f2f74b1579fa79441f7ea18ed128876f0f37) is to be reviewed here. Two others are from #1107 with which I would conflict with.

Reply:
Fixed in #1107.